src/domain/reservation/manage/page/ManageReservationsPage.js

```diff
@@ -13,6 +15,9 @@ import ManageReservationsList from '../list/ManageReservationsList';
 import Pagination from '../../../../common/pagination/Pagination';
 import * as searchUtils from '../../../search/utils';
 import ReservationInfomationModal from '../../modal/ReservationInfomationModal';
+import { selectReservationToEdit } from '../../../../../app/actions/uiActions';
```

Comment:
I'd rather do without this action

Reply:
The edit reservation view need this to work, they still use redux